doc/contributing/collaborator-guide.md

```diff
@@ -154,10 +154,13 @@ requirements. If a pull request meets all requirements except the
 
 Collaborators can object to a pull request by using the "Request
 Changes" GitHub feature. Dissent comments alone don't constitute an
-objection. Any pull request objection must include a clear reason for that
-objection, and the objector must remain responsive for further discussion
-towards consensus about the direction of the pull request. Where possible,
-provide a set of actionable steps alongside the objection.
+objection, nor do dissenting comments made in any associated issue.
+A blocking objection to a change must be made in the pull request that
+specifically proposes that change. Any pull request objection must include
+a clear reason for that objection, and the objector must remain responsive
+for further discussion towards consensus about the direction of the pull
+request. Where possible, provide a set of actionable steps alongside the
+objection.
```

Comment:
```suggestion
objection.
If a person objected in an issue, or a former PR, please mention them in the PR description
and provide up to a week for them to reply.
```

That way we prevent someone to just go ahead and open a new PR and someone missing the new one.
I think that would be beneficial.

Reply:
I don't think it's reasonable to add that, it's too easy to miss. Pinging possible objectors is a nice thing to do, but we cannot have it as a rule, that's the reason we added the "objections must be done through GH UI" rule

Reply:
Agreed. blocks need to be explicit or they are way too easy to miss.

thank you for the suggested edit but I prefer the current text over the suggested edit.